### ISSUE TITLE
Grouped data: information about column types should be removed

### ISSUE THREAD

**Issue by GegznaV** [ropensci/skimr#464]
The output of **grouped data** contains an unnecessary row with `<chr>` etc.
Is it a regression as this issue seemed to be solved in  #384?

From the examples of branch v2: https://github.com/ropensci/skimr/tree/v2
```r
iris %>%
    dplyr::group_by(Species) %>%
    skim()
```

![image](https://user-images.githubusercontent.com/12725868/59160462-cdb3db80-8ade-11e9-9b9d-72d9b2dfb6a9.png)
![image](https://user-images.githubusercontent.com/12725868/59160595-33549780-8ae0-11e9-8de2-1864e76dc417.png)

Possibly related to #376, #384, #408.

**Comment by elinw** [comment]
Sigh, this is the tibble issue again.  I think we probably just missed dealing with it in the grouped case. Thanks for reporting.

**Comment by elinw** [comment]
So I've been exploring today I haven't been able to find the pattern yet.  mtcars prints the metadata all the time, CO2 never, iris when it's grouped but not ungrouped.

**Comment by GegznaV** [comment]
Does this behavior depend on the initial class(es) of an object that is passed to `skim()`? Maybe the S3 system does not dispatch the methods correctly?

``` r
class(mtcars) 
#> [1] "data.frame"

class(CO2) 
#> [1] "nfnGroupedData" "nfGroupedData"  "groupedData"    "data.frame"

class(iris) 
#> [1] "data.frame"

class(dplyr::group_by(iris, Species)) 
#> [1] "grouped_df" "tbl_df"     "tbl"        "data.frame"
```

<sup>Created on 2019-06-21 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by elinw** [comment]
Oh wow, that is intriguing.  For CO2 I get different results depending on if I group by Treatment, conc or Plant.

**Comment by elinw** [comment]
This was closed by #476
